### ISSUE TITLE
Is ACME's `renewalInfo` on the plan list?

### ISSUE THREAD

**Issue by ldlb9527** [certbot/certbot#9986]
https://datatracker.ietf.org/doc/html/draft-ietf-acme-ari-03#name-renewalinfo-objects

**Comment by osirisinferi** [comment]
The `renewalInfo` directory entry is just a part of the overarching entity called "ACME Renewal Information" (ARI), which that draft RFC is all about, and is currently developed in https://github.com/certbot/certbot/pull/9945 by a third party contributor.

**Comment by zoracon** [comment]
Yes it is on the plan list for 2025. Thank you for bringing this up and we will have proper updates in the community forum and here when it is ready to be released in Certbot.
